packages/ui/src/components/style/core/global.less

```diff
@@ -6,4 +6,11 @@
 :root {
   --condo-font-fallback: sans-serif;
   --condo-font-fallback-mono: consolas, monaco, "Andale Mono", "Ubuntu Mono", monospace;
-}
\ No newline at end of file
+}
+
+@condo-global-breakpoints-mobile-small: 0px;
```

Comment:
It's not usable like that....

You can use a vars like that
<img width="734" alt="image" src="https://user-images.githubusercontent.com/50069872/230209288-00fb0464-0f06-43f0-97fb-c062102632d1.png">


Or use a mixins:
![image](https://user-images.githubusercontent.com/50069872/230209532-e6b83cbe-2804-4504-931f-49d8574c0b57.png)